### PR TITLE
Miscellaneous refactoring

### PR DESCRIPTION
**Pull request by M4rtinK**
Miscellaneous refactoring and style fixes based mostly on output from PEP8, Pyflakes, Pylint and other checkers.

### CODE REVIEW

File: pyanaconda/bootloader.py [hunk 1]
```diff
@@ -1362,16 +1361,16 @@ def is_valid_stage2_device(self, device, linux=True, non_linux=False):
                 self.stage2_device.level in self.stage2_raid_levels and \
                 self.stage1_device.type != "mdarray":
             if not self.stage1_device.is_disk:
-                msg = _("boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot loader stage1 device %(stage1dev)s is not. " \
+                msg = _("boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot loader stage1 device %(stage1dev)s is not. "
```
Comment (jkonecny12) [inline pyanaconda/bootloader.py]: Could you please wrap this line?

File: pyanaconda/bootloader.py [hunk 2]
```diff
@@ -1362,16 +1361,16 @@ def is_valid_stage2_device(self, device, linux=True, non_linux=False):
...
                         "A drive failure in %(stage2dev)s could render the system unbootable.") % \
-                        {"stage1dev" : self.stage1_device.name,
-                         "stage2dev" : self.stage2_device.name}
+                        {"stage1dev": self.stage1_device.name,
+                         "stage2dev": self.stage2_device.name}
                 self.warnings.append(msg)
             elif not self.stage2_device.depends_on(self.stage1_device):
-                msg = _("boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot loader stage1 device %(stage1dev)s is not part of this array. " \
+                msg = _("boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot loader stage1 device %(stage1dev)s is not part of this array. "
```
Comment (jkonecny12) [inline pyanaconda/bootloader.py]: Same here, could you please wrap this?

File: pyanaconda/bootloader.py (hunk 1 above)
Comment (M4rtinK) [inline pyanaconda/bootloader.py]: Fixed.

File: pyanaconda/bootloader.py (hunk 2 above)
Comment (M4rtinK) [inline pyanaconda/bootloader.py]: Fixed as well. :)

**Comment by poncovka** [comment]
Looks good to me. Thanks!

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
I would like to run tests on this but we are experiencing problems with repositories. Our COPR repository behaves weird.

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
jenkins, test this please

**Pull request merged**